From 79f02a53316f90543d60269d7c06727c376f423b Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Wed, 24 Aug 2022 18:41:08 +0000
Subject: [PATCH 1/1] fix-sdk-download

Sdk insists on downloading SDK rather than using previously unpacked ones.
This patch reverts to old behavior

---
diff --git a/src/sdk/eng/restore-toolset.sh b/src/sdk/eng/restore-toolset.sh
index 2426dac65b0..9e3f38f8536 100644
--- a/src/sdk/eng/restore-toolset.sh
+++ b/src/sdk/eng/restore-toolset.sh
@@ -7,7 +7,7 @@ function InitializeCustomSDKToolset {
 
   # The following frameworks and tools are used only for testing.
   # Do not attempt to install them in source build.
-  if [[ $properties == *"ArcadeBuildFromSource=true"* ]]; then
+  if [[ "${DotNetBuildFromSource:-}" == "true" ]]; then
     return
   fi
 
